rtloader/three/three.cpp

```diff
@@ -395,6 +395,41 @@ bool Three::getCheck(RtLoaderPyObject *py_class, const char *init_config_str, co
     return true;
 }
 
+bool Three::isCheckInitDeprecated(RtLoaderPyObject *py_class)
+{
+    PyObject *klass = reinterpret_cast<PyObject *>(py_class);
+    PyObject *init, *func_code, *co_varnames, *elt;
```

Comment:
Please init those variables. I know we init them before using them but if will make this function future proof (we had the issue like this before on RTLoader).

Reply:
Done.